gateway/gateway-service-api.yaml

```diff
@@ -67,27 +67,20 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
         '415':
-          description: Media type other than application/json provided in body
-          content:
-            text/plain:
-              schema:
-                type: string
+          $ref: '#/components/responses/ContentTypeNotJson'
         '500':
           $ref: '#/components/responses/UnexpectedError'
   /api/v1/gateway/users/{userUuid}:
     get:
       summary: Gets the basic account information for the given user.
-      description: This will return the basic account information for a user of the system. Requires the client to be authenticated. Currently any user can get the basic account information for any other user, this may change in the future.
+      description: This will return the basic account information for a user of the system. Requires the client to be authenticated. Currently any user can get the basic account information for any other user, this may change in the future. (Authorization header required, see '#/components/headers/Authorization')
```

Comment:
I'm surprised since we had all talked about defaulting to privacy, but I understand that may take more work, and we only have so much time (and since the AKS cluster hosting this wont usually be up, I can't really see much justification to focusing on it right now).

Reply:
I was debating making that open to all authenticated users. The main reason being that call only returns: username, display name, full name, and uuid, so this was meant to be a way for say, one user to see basic information about another users content (so for example, if there is a list of modules/quizes and each one was created by a different individuals content, if we wanted to display who added it there would be a way to get basic information based on the users uuid, without having to store a copy of that information with whatever item was created by the user.)

@kml27, given that information do you think this request should be restricted so that the authenticated user can only get information about themself?